defs.bzl

```diff
@@ -28,10 +42,14 @@ pip_repository = repository_rule(
         "wheel_env": attr.string_dict(),
         "python_interpreter": attr.string(default="python3"),
         "_script": attr.label(
-            executable=True,
-            default=Label("//tools:wheel_wrapper.py"),
-            cfg="host",
+            executable=True, default=Label("//src:__main__.py"), cfg="host",
         ),
     },
     implementation=_pip_repository_impl,
 )
+
+
+def pip_install(requirements, name=DEFAULT_REPOSITORY_NAME):
```

Comment:
pip_install is now a macro. It is significantly more flexible. This enables us to add more repository rules when we go to implement the prebuilt wheels.